airbyte-integrations/connectors/source-primetric/source_primetric/spec.yaml

```diff
@@ -0,0 +1,24 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Primetric Spec
+  type: object
+  required:
+    - client_id
+    - client_secret
+  properties:
+    client_secret:
+      type: string
+      title: Client Secret
+      description: The Client Secret of your Primetric developer application.
```

Comment:
Can we add a link to the account's client credentials page, similar to what is done [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-stripe/source_stripe/spec.yaml#L15) ?